overrides/resources/pack.mcmeta

```diff
@@ -1 +1 @@
-{"pack":{"pack_format":3,"description":"Omnifactory Resources"}}
\ No newline at end of file
+{"pack":{"pack_format":3,"description":"B.A.S.E External Resources"}}
```

Comment:
?

Reply:
That's BASE being stupid, I suppose it ought to be done away with.